### PR TITLE
Adds autorefresh with play/pause

### PR DESCRIPTION
**Pull request by bthesorceror**
- [x] Rebased/mergeable
  - [ ] Tests pass
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

**Comment by alexpaxton** [comment]
screen shot 2018-06-01 at 4 59 08 pm

File: ui/src/logs/components/LogsTable.tsx [hunk 1]
```diff
@@ -1,39 +1,64 @@
...
-const ROW_HEIGHT = 40
+const ROW_HEIGHT = 30
+const HIGHLIGHT_COLOR = '#555'
 
 interface Props {
   data: {
     columns: string[]
     values: string[]
   }
+  scrolledToTop: boolean
```
Comment (ischolten) [inline ui/src/logs/components/LogsTable.tsx]: there was talk about prefixing boolean variables with `is`, how do you feel about making that change here?

File: ui/src/logs/components/LogsTable.tsx [hunk 2]
```diff
@@ -116,7 +147,7 @@ class LogsTable extends PureComponent<Props, State> {
 
     switch (column) {
       case 'message':
-        return 900
+        return 1200
```
Comment (ischolten) [inline ui/src/logs/components/LogsTable.tsx]: consider making these constants or enums

File: ui/src/logs/components/LogsTable.tsx [hunk 3]
```diff
@@ -181,22 +214,39 @@ class LogsTable extends PureComponent<Props, State> {
...
+        onMouseOver={this.handleMouseOver}
+        data-index={rowIndex}
+      >
         {value}
       </div>
     )
   }
+
+  private handleMouseOver = (e: MouseEvent<HTMLElement>) => {
+    const target = e.target as HTMLElement
```
Comment (ischolten) [inline ui/src/logs/components/LogsTable.tsx]: is there a reason you need to cast this as HTMLElement?

File: ui/src/logs/containers/LogsPage.tsx [hunk 4]
```diff
@@ -51,25 +51,29 @@ interface Props {
...
 
 const DUMMY_FILTERS = [
-  {
-    id: '0',
-    key: 'host',
-    value: 'prod1-rsavage.local',
-    operator: '==',
-    enabled: true,
-  },
+  // {
```
Comment (ischolten) [inline ui/src/logs/containers/LogsPage.tsx]: could you delete this commented code?

Comment (bthesorceror) [reply ui/src/logs/containers/LogsPage.tsx]: I had commented this out just to get a feeling of how it looks but can uncomment.

File: ui/src/logs/components/LogsTable.tsx (hunk 3 above)
Comment (bthesorceror) [reply ui/src/logs/components/LogsTable.tsx]: yeah, I had an issue with accessing the dataset that was fixed by casting. Going to revisit on the next chunk of work for this feature set.

File: ui/src/logs/components/LogsTable.tsx (hunk 2 above)
Comment (bthesorceror) [reply ui/src/logs/components/LogsTable.tsx]: Agree they should be constants.

File: ui/src/logs/components/LogsTable.tsx (hunk 1 above)
Comment (bthesorceror) [reply ui/src/logs/components/LogsTable.tsx]: 👍

File: ui/src/logs/components/LogViewerHeader.tsx [hunk 5]
```diff
@@ -29,7 +32,10 @@ class LogViewerHeader extends PureComponent<Props> {
       <div className="page-header full-width">
         <div className="page-header__container">
           <div className="page-header__left">
-            <h1 className="page-header__title">Log Viewer</h1>
+            {this.status}
+            <h1 className="page-header__title" style={{marginLeft: '10px'}}>
```
[the remark this answers is not in the document]
Comment (bthesorceror) [reply ui/src/logs/components/LogViewerHeader.tsx]: 👍

**Pull request merged**